webapp/channels/src/components/global_header/left_controls/product_menu/product_branding_team_edition/product_branding_free_edition.tsx

```diff
@@ -36,16 +41,25 @@ const Badge = styled.span`
     line-height: 16px;
 `;
 
-const ProductBrandingTeamEdition = (): JSX.Element => {
+const ProductBrandingFreeEdition = (): JSX.Element => {
+    const license = useSelector(getLicense);
+
+    let badgeText = '';
+    if (license?.SkuShortName === LicenseSkus.Entry) {
+        badgeText = 'ENTRY EDITION';
+    } else if (license?.IsLicensed === 'false') {
+        badgeText = 'FREE EDITION';
```

Comment:
```suggestion
        badgeText = 'TEAM EDITION';
```

Reply:
Missed that from the spec, nice catch! Updated: 

<img width="329" height="486" alt="Screenshot 2025-09-08 at 11 23 26 AM" src="https://github.com/user-attachments/assets/3002c8c2-ffa1-49a9-8a89-8e2f9b5a3773" />